frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -384,8 +404,7 @@
       "name": "default",
       "model": {
         "group": "kubevirt.io",
-        "kind": "VirtualMachine",
-        "version": "v1alpha3"
+        "kind": "VirtualMachine"
```

Comment:
I'd prefer to contribute separate templates per version.
When we adopt typescript definitions of the console extensions, you can reuse the same function to generate the the yaml.